spec/models/comment_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
```

Comment:
- [ ] All the methods in the `Comment` Model should be tested, kindly make sure to implement unit tests for the `update_comments_counter` method